commands/create/kubeconfig/command.go

```diff
@@ -354,6 +359,40 @@ func createKubeconfigRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 	}
 }
 
+// getClusterDetails fetches cluster details to get the tenant cluster API endpoint,
+// and attempts first v5 and then falls back to v4.
+func getClusterDetails(clientWrapper *client.Wrapper, clusterID string, auxParams *client.AuxiliaryParams, verbose bool) (string, error) {
+	// Try v5 first, then fall back to v4.
+	if verbose {
+		fmt.Println(color.WhiteString("Fetching cluster details using the v5 API endpoint"))
+	}
+	clusterDetailsResponseV5, err := clientWrapper.GetClusterV5(clusterID, auxParams)
+	if err != nil {
+		if clienterror.IsNotFoundError(err) {
+			// We ignore a 404 here and try v4 next.
+			if verbose {
+				fmt.Println(color.WhiteString("Cluster not found via the v5 endpoint. Attempting v4 endpoint."))
+			}
+			clusterDetailsResponseV4, err := clientWrapper.GetClusterV4(clusterID, auxParams)
+			if err != nil {
+				if clientErr, ok := err.(*clienterror.APIError); ok {
+					return "", microerror.Maskf(clientErr,
+						fmt.Sprintf("HTTP Status: %d, %s", clientErr.HTTPStatusCode, clientErr.ErrorMessage))
```

Comment:
```suggestion
						"HTTP Status: %d, %s", clientErr.HTTPStatusCode, clientErr.ErrorMessage)
```
I probably missed this earlier, but with `Maskf()` `fmt.Sprintf()` is not needed. `f` in `Maskf` means formatting :)